vcloud-RHEL-manager-blueprint-inputs.yaml

```diff
@@ -1,5 +1,5 @@
 #############################
-# vCloud Air specific Inputs
+# vCloud specific Inputs
```

Comment:
the idea was that each input in the input files will have an explanation and that WHATEVER can have a default, will have a default so that users can configure as less as possible before bootstrapping.
Please add the descriptions and defaults where possible.
Thanks